packages/governance-pool/contracts/mock/FakeInvestmentPoolMock.sol

```diff
@@ -0,0 +1,23 @@
+// @ buidl.one 2022
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.9;
+
+import {FakeInvestmentPoolMock} from "@buidlone/investment-pool/contracts/mock/FakeInvestmentPoolMock.sol";
+import {IGovernancePool} from "@buidlone/investment-pool/contracts/interfaces/IGovernancePool.sol";
+
+contract FakeInvestmentPoolMockV2 is FakeInvestmentPoolMock {
```

Comment:
V2 in the name of a mock is kind of ambiguous.
Perhaps there is a more fitting one?